### PR TITLE
Refactored some messages, code structure and params

### PR DESCRIPTION
**Pull request by CosminNechifor**
Those changes were performed only for consistency reasons

I've changed the messages of this script since is kinda confusing when it's being called from ``wcm-system`` (if the cleanup option was being provided to the scripts). Also I've done some changes to the way it was written since some of the lines were 150+ chars which was kinda hard to follow.

Related PR: cisco-app-networking/wcm-system#124
Closes: cisco-app-networking/cnns-prj#115

### CODE REVIEW

File: scripts/vl3/vl3_interdomain.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ for i in "$@"; do
         --wcmNsrPort=?*)
             WCM_NSRPORT=${i#*=}
             ;;
-        --delete)
+        --cleanup)
```
Comment (tiswanso) [inline scripts/vl3/vl3_interdomain.sh]: In the interest of not breaking existing usage of this script, changes to params should be additive rather than removing/replacing existing params.  So, make this `--delete|--cleanup)` so either will work.

Comment (CosminNechifor) [reply scripts/vl3/vl3_interdomain.sh]: Ok I'll do that.

**Review by danielvladco — COMMENTED** [review]
I think it's a good idea to to not ignore the `kubectl wait` and wait for delete instead

**Comment by CosminNechifor** [comment]
> I think it's a good idea to to not ignore the `kubectl wait` and wait for delete instead

I will not change it in this PR. I think it can be added as a ``good first issue``. Will allow anyone that joins to take a look at the code and see how the installation/cleanup of the topology is being done. Do you agree? 

If you guys consider that this should also be added together with the ``cleanup`` changes that were made on the repos, then please open a issue and assign me to it.

**Review by danielvladco — APPROVED** [review]

**Review by tiswanso — APPROVED** [review]
